.travis.yml

```diff
@@ -6,14 +6,19 @@ services:
   - docker
 
 before_script:
+  - export PROJECT_DIR=$PWD
   - echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+  - cd $PROJECT_DIR/api
+  - npm i
```

Comment:
npm install commands should be in install section of the travis file and since we are now installing node_modules in raw travis VM we should make sure the node_modules are cached.
https://docs.travis-ci.com/user/customizing-the-build#Customizing-the-Installation-Step
https://docs.travis-ci.com/user/caching/#Arbitrary-directories